packages/modal-ui/src/lib/components/WalletOptions.tsx

```diff
@@ -29,6 +29,13 @@ export const WalletOptions: React.FC<WalletOptionsProps> = ({
 
   useEffect(() => {
     const subscription = selector.store.observable.subscribe((state) => {
+      state.modules.sort((current, next) => {
+        return current.metadata.deprecated === next.metadata.deprecated
+          ? 0
+          : current.metadata.deprecated
+          ? 1
+          : -1;
+      });
```

Comment:
Multiple ternary operators can be difficult to read. Can we change this to:
```suggestion
      state.modules.sort((current, next) => {
        if (current.metadata.deprecated === next.metadata.deprecated) {
          return 0;
        }
        
        return current.metadata.deprecated ? 1 : -1;
      });
```